src/mixins/input.js

```diff
@@ -63,6 +63,9 @@ export default {
         'theme--light': this.light
       }, this.classes)
     },
+    isClearable () {
+      return this.disabled ? false : this.clearable
```

Comment:
`return this.clearable && !this.disabled`

Reply:
No problem, will include.
I did npm install so lockfile got updated.
Should I revert lock file or will you handle that on merge?

Reply:
Yes, remove it please, you can push --force